corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -70,8 +70,7 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None,
                             unique_form_id=form.unique_id)
 
     vellum_plugins = ["modeliteration", "itemset"]
-    if (toggles.VELLUM_TRANSACTION_QUESTION_TYPES.enabled(domain) or
-            toggles.COMMTRACK.enabled(domain)):
+    if (toggles.COMMTRACK.enabled(domain)):
```

Comment:
doesn't look like it

For future reference you can view the API between HQ and vellum here: https://github.com/dimagi/Vellum/blob/master/tests/options.js